simple-facebook-instant-articles.php

```diff
@@ -74,6 +78,34 @@ public function __construct( $file, $version ) {
 		$this->template_path = trailingslashit( $this->dir ) . 'templates/';
 		$this->home_url      = trailingslashit( home_url() );
 		$this->endpoint      = apply_filters( 'simple_fb_article_endpoint', 'fb-instant' );
+		$this->options       = get_option( 'fb_instant' );
+	}
+
+	/**
+	 * The register activation hook should flush the rewrite rules, in the event
+	 * that it doesn't let's go ahead and flush the rules.
+	 *
+	 * @return void
+	 */
+	public function flush_rules() {
```

Comment:
There has been a lot of feedback feeds don't work when a person activates the plugin. This ensures the manual step of updating permalinks. Is there a conflict on VIP or elsewhere?